modules/tinymce/src/plugins/lists/main/ts/core/ListNumbering.ts

```diff
@@ -45,7 +45,7 @@ const isLowercase = (str: string): boolean => /^[a-z]+$/.test(str);
 const isNumeric = (str: string): boolean => /^[0-9]+$/.test(str);
 
 const deduceListType = (start: string): ListType => {
-  if (isNumeric(start)) {
+  if (isNumeric(start) || start === '') {
```

Comment:
Not for this PR, but I wonder if an empty string check should be a simple function in Katamari?

Reply:
It already exists btw... `Strings.isEmpty`

Reply:
Ahh I looked in the wrong place!